staging/src/k8s.io/apiserver/pkg/util/webhook/validation.go

```diff
@@ -28,6 +29,10 @@ import (
 
 func ValidateCABundle(fldPath *field.Path, caBundle []byte) field.ErrorList {
 	var allErrors field.ErrorList
+	// Treat whitespace-only caBundle as empty
```

Comment:
I don't think we really want to relax validation here ... this is what keeps us from allowing new invalid data into CRDs, and keeps us from establishing CRDs with webhooks configured with CABundles that won't work properly